src/Components/CommentsList.jsx

```diff
@@ -0,0 +1,44 @@
+import { useState, useEffect } from "react";
+import { getCommentsByArticleId } from "../Utils/api";
+import { useParams } from "react-router-dom";
+import CommentCard from "./CommentCard";
+
+export default function CommentsList() {
+  const [commentsData, setCommentsData] = useState(null);
+  const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState(false);
+  const { article_id } = useParams();
+
+  useEffect(() => {
+    getCommentsByArticleId(article_id)
+      .then(({ data }) => {
+        setCommentsData(data.data);
+        setIsLoading(false);
+      })
+      .catch(({ response }) => {
+        setError(response);
+        setIsLoading(false);
+      });
+  }, []);
+  if (error)
+    return (
+      <h2>
+        {error.status} : {error.data.message}
+      </h2>
+    );
+
+  return (
+    <>
+      <h2 className="article-list-header">-Comments-</h2>
+      <section className="comment-list">
+        {isLoading ? (
+          <h2>Loading....</h2>
+        ) : (
+          commentsData.map((comment) => {
+            return <CommentCard key={comment.comment_id} comment={comment} />;
+          })
+        )}
+      </section>
+    </>
+  );
```

Comment:
Have a think about what you want to display if there's no comments on an article (for example if you check article id 37, it has no associated comments yet).

At the moment - it shows a blank comment area, maybe a little message saying 'no comments yet' may look better?